src/Microsoft.DotNet.Interactive/Utility/Dotnet.cs

```diff
@@ -21,14 +23,19 @@ public Dotnet(DirectoryInfo workingDirectory = null)
                             new DirectoryInfo(Directory.GetCurrentDirectory());
     }
 
-    public Task<CommandLineResult> New(string templateName, string args = null)
+    public async Task<CommandLineResult> New(string templateName, string args = null)
     {
         if (string.IsNullOrWhiteSpace(templateName))
         {
             throw new ArgumentException("Value cannot be null or whitespace.", nameof(templateName));
         }
 
-        return Execute($@"new ""{templateName}"" {args}");
+        return await Execute($@"new ""{templateName}"" {args}");
+    }
+
+    public Task<CommandLineResult> NewGlobalJson()
```

Comment:
As long as this class is public, we should try to avoid adding new public methods.

Reply:
removed